### ISSUE TITLE
Crash when loading after config (synchro)

### ISSUE THREAD

**Issue by eboustou** [RphWbr/visiannot#13]
```
Traceback (most recent call last):
  File "C:\Users\Edouard\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Edouard\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Edouard\git\visiannot\visiannot\__main__.py", line 109, in <module>
    win_visi = ViSiAnnoTLongRecFromConfigGUI(path=config_path)
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoTLongRecFromConfigGUI.py", line 37, in __init__
    ViSiAnnoTLongRecFromConfigFile.__init__(self, config_dict)
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoTLongRecFromConfigFile.py", line 70, in __init__
    ViSiAnnoTLongRec.__init__(self, *args, **kwargs)
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoTLongRec.py", line 366, in __init__
    ViSiAnnoT.__init__(
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoT.py", line 485, in __init__
    self.setAllData(video_dict, signal_dict, interval_dict)
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoT.py", line 2047, in setAllData
    data, freq_data = self.getDataSigTmp(
  File "C:\Users\Edouard\git\visiannot\visiannot\visiannot\ViSiAnnoT.py", line 2271, in getDataSigTmp
    next_data_ts <= temporal_limit
UnboundLocalError: local variable 'next_data_ts' referenced before assignment
```

**Comment by RphWbr** [comment]
I will need more info: configuration file, data file involved.

It should not crash because next_data_ts is defined in the "if" loop at line 2214 in case the data is not regularly sampled (2D array with timestamps). Then next_data_ts is only used in "if" loops with the same condition.

**Comment by RphWbr** [comment]
Bug fixed in commit #5ee8d881412de05a57049d0a5a39c1afba489f8d
